src/util.h

```diff
@@ -41,6 +42,16 @@
 #include <unordered_map>
 #include <utility>
 
+// Maybe remove kPathSeparator when cpp17 is ready
+#ifdef _WIN32
+const char* const kPathSeparator = "\\/";
```

Comment:
This will result in duplicate definition build errors when more than one source file uses `kPathSeparator`. The actual value is wrong too, it should be `"\\"`. Copy/paste error, I presume? :-)

I think you can make this `constexpr char kPathSeparator[] = "\\";` but since it's just a single character, I'd make it `char`, not a `char` array.

And please `namespace node {` it.

Reply:
> Copy/paste error, I presume? :-)
Yeap, directly copied from `inspector_profiler.cc`

Reply:
@bnoordhuis Can you review this again ? thanks.